### PR TITLE
chore(ci): Mergify - merge Autobumps on release-*

### PR DESCRIPTION
**Pull request by kskewes-sf**
And request review for failed autobump builds on master and release
branches.

Lastly, remove Travis CI related rule. It's not used.

### CODE REVIEW

File: .mergify.yml [hunk 1]
```diff
@@ -56,7 +42,7 @@ pull_request_rules:
         add: ["auto merged"]
   - name: Automatically merge autobump PRs on CI success
     conditions:
-      - base=master
+      - base~=^(master|release-[0-9]+.[0-9]+.x)$
```
Comment (kskewes-sf) [inline .mergify.yml:45]: This could be simpler `- base~=^(master|release-)` perhaps per rule beginning L19..?

**Review by dbyron-sf — APPROVED** [review]

File: .mergify.yml (hunk 1 above)
Comment (dbyron-sf) [reply .mergify.yml]: Yup, agreed.  https://docs.mergify.com/configuration/#regular-expressions

**Review by dbyron-sf — APPROVED** [review]

**Comment by dbyron-sf** [comment]
@backport mergifyio release-1.27.x release-1.28.x

**Comment by dbyron-sf** [comment]
@mergifyio backport release-1.27.x release-1.28.x

**Pull request merged**